docs/APIRef.DeviceObjectAPI.md

```diff
@@ -317,11 +316,14 @@ Takes `"portrait"` or `"landscape"` and rotates the device to the given orientat
 
 Check out Detox's [own test suite.](../detox/test/e2e/06.device-orientation.test.js)
 
-### `device.setLocation(lat, lon)` **iOS Only**
+### `device.setLocation(lat, lon)`
 
-> Note: `setLocation` is dependent on [fbsimctl](https://github.com/facebook/idb/tree/4b7929480c3c0f158f33f78a5b802c1d0e7030d2/fbsimctl) which [is now deprecated](https://github.com/wix/Detox/issues/1371). If `fbsimctl` is not installed, the command will fail, asking for it to be installed.
+> Note: 
+> On iOS `setLocation` is dependent on [fbsimctl](https://github.com/facebook/idb/tree/4b7929480c3c0f158f33f78a5b802c1d0e7030d2/fbsimctl) which [is now deprecated](https://github.com/wix/Detox/issues/1371). If `fbsimctl` is not installed, the command will fail, asking for it to be installed.
+> 
+> On Android `setLocation` will work with both Android Emulator (bundled with Android development tools) and Genymotion. The correct permissions must be set in your app manifest.
```

Comment:
@emilisb How come we didn't have to add this in our test apps?

Reply:
It was already there https://github.com/wix/Detox/blob/8c0e4b530e7cb326117f4d64cb56f0a1ee2392d6/detox/test/android/app/src/main/AndroidManifest.xml#L10

Reply:
Go figure... Anyways, thanks again 😄